lib/V8/v8-utils.cpp

```diff
@@ -202,6 +202,85 @@ static void CreateErrorObject(v8::Isolate* isolate, int errorNumber,
   }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief set a point in time after which we will abort external connection
+////////////////////////////////////////////////////////////////////////////////
+static std::atomic<double> executionDeadline(0.0);
```

Comment:
Is there just a single thread?

Reply:
in arangosh - yes. however, it should be correct for arangod.

Reply:
I am not sure. Why should it be correct to have a single variable for potentially multiple contexts?